content/en/blog/2022/collector-builder-sample/index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Using the Collector Builder with Sample Configs on GCP
+linkTitle: Collector builder sample with GCP
+date: 2022-10-03 # Put the current date, we will keep the date updated until your PR is merged
+author: >-
+  [Mike Dame](https://github.com/damemi) (Google)
+---
+
+The [OpenTelemetry Collector](/docs/collector/) is a versatile tool for
+processing and exporting telemetry data. It supports ingestion from many
+different sources and routing to just as many observability backends thanks to
+its modular design. This design, based on individual receivers, processors, and
+exporters, enables third-party development of new backend support. It also
+allows users to configure telemetry pipelines that are customized to their use
+case.
+
+The
+[collector-builder tool](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder)
+takes that customizability a step further, offering a way to easily compile a
+Collector binary built with only certain ingestion and export components. In
+contrast with publicly available binary releases (which bundle in a number of
+components by default), custom Collectors can be slimmed down to only include
+the components you really care about. This means the compiled Collector can be
+smaller, faster, and more secure than a public release.
+
+Google Cloud recently launched a
+[sample GitHub repository](https://github.com/GoogleCloudPlatform/opentelemetry-collector-builder-sample)
+for building and deploying your own custom OpenTelemetry Collector on GCP. This
+takes the guesswork out of running the collector on GCP, but leaves you free to
+extend it to meet your use-cases.
+
+This repository will help you:
+
+- **Compile a custom Collector designed for GCP with the**
+  [**collector-builder**](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder).
+  Low-overhead Collector builds are possible with the upstream collector-builder
+  tool, so we’ve provided the setup files for building a lightweight collector
+  with GCP services in mind. This is GCP's recommended base set of components to
+  use when running on GCP or working with GCP services. OpenTelemetry can be
```

Comment:
@svrnm would you prefer to cut out the GCP services mentioned here?

Reply:
This sounds to me as it is referring to "GCP the infrastructure", so let's keep it :-)